docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -390,9 +505,13 @@ Now all we have to do is create a vertex buffer (don't worry it's really easy) a
 
 ## Vertex Buffers
 
-Vertex Buffers might seem hard at first, but they're really nothing more than a buffer that resides in our device's memory, which means really fast access. This buffer will be then bound and read by the vertex shader.
+Vertex Buffers might seem hard at first, but they're really nothing more than a 
+buffer that resides in our device's memory, which means really fast access. This 
+buffer will be then bound and read by the vertex shader.
 
-Creating a vertex buffer is also really easy, first we have to make some data to put in our buffer, since we want to draw a triangle, we will be creating 3 vertices using our `VertexPositionColor` struct.
+Creating a vertex buffer is also really easy, first we have to make some data to put 
+in our buffer, since we want to draw a triangle, we will be creating 3 vertices using 
+our `VertexPositionColor` struct.
 
 ```cpp
 constexpr VertexPositionColor vertices[] =
```

Comment:
use std::array instead? also gets rid of this atrocious `_countof` which I don't believe is portable at all?  (because we can use `.size()` then)

Reply:
If this is changed in the docs, it needs to be changed in code as well.

Mhh not sure how much the comments divert from the actual scope of this PR. This should be addressed in a different issue/PR